RELEASE.md

```diff
@@ -6,30 +6,46 @@ Official Falcosecurity rules releases are automated using GitHub Actions. Each r
 
 In this repo, each ruleset is a standalone YAML file in the `/rules` directory (e.g. `falco_rules.yaml`, `application_rules.yaml`, ...). Each ruleset is released and versioned individually. When we release a ruleset, we do the following process:
 
-1. Determine a new version for the given ruleset (see the [section below](#versioning-a-ruleset))
-2. Create a new Git tag with the name convention `*name*-rules-*version*` (e.g. `falco-rules-0.1.0`, `application-rules-0.1.0`, ...). The naming convention is required due to this repository being a [monorepo](https://en.wikipedia.org/wiki/Monorepo) and in order to be machine-readable.
-3. A GitHub action will validate the repository [registry](./registry.yaml) and release the new OCI artifact in the packages of this repository
+1. Make sure that the `./github/FALCO_VERSIONS` file contains the most recent versions of Falco compatible with the given ruleset. The `master` version must not be used when releasing a ruleset.
```

Comment:
Are we implicitly saying releases can only happen on a release branch? :thinking:

Reply:
Nope. `master` is just a valid Falco version keyword, that can be used to run the latest dev version of Falco in the CI. It has nothing to do with how we manage the ruleset releases.

The message here is that `master` can be used when we need to leverage a not-yet-release version of Falco to run the CI checks for any given reason (maybe a new feature). However, when a ruleset gets tagged it would be preferable to point to a stable released version of Falco. This could apply if either we decide to have branch releases or not in the future.